.changeset/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "changelog": ["@changesets/changelog-github", { "repo": "atlassian/extract-react-types" }],
```

Comment:
@Noviny, i feel like we need to move this somewhere 😨

Reply:
Oh interesting. I think using the GH changelog function is probably better. Basically you can have a local one written, or you can use one of our defaults, the GH one being useful here.

If you don't want to worry. You could also use the simple [git changelog generator](https://github.com/atlassian/changesets/tree/master/packages/changelog-git)

Basically, if you pass a package name, it assumes the package exports the required files and does that.